CefSharp/IWebBrowser.cs

```diff
@@ -134,6 +134,11 @@ public interface IWebBrowser : IChromiumWebBrowserBase
         /// </summary>
         IFrameHandler FrameHandler { get; set; }
 
+        /// <summary>
+        /// Implement <see cref="IPermissionHandler" /> to handle permission prompt.
```

Comment:
Update to match CEF wording.

https://github.com/chromiumembedded/cef/blob/master/include/cef_permission_handler.h#L83